e2e/manifests/overlays/daemonset-lvmd/controller.yaml

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
I recognize this file is the same as common/controller.yaml except for node selector.
Kustomize supports deletion for yaml fragment, using `$delete` is clean manner, IMO.
see https://github.com/kubernetes-sigs/kustomize/blob/master/examples/inlinePatch.md